src/main/webapp/data-visualization.js

```diff
@@ -508,6 +529,142 @@ function toggleDataTable() {
   }
 }
 
+function displayYearlyChart(chartId, data, description, location) {
+  const chart = am4core.create(chartId, am4charts.XYChart);
+  chart.data = data;
+  const dateAxis = chart.xAxes.push(new am4charts.DateAxis());
+  dateAxis.dataFields.category = 'year';
+  dateAxis.title.text = 'Year';
+  dateAxis.baseInterval = {timeUnit: 'year'};
+  const valueAxis = chart.yAxes.push(new am4charts.ValueAxis());
+  valueAxis.title.text = description;
+  const series = chart.series.push(new am4charts.LineSeries());
+  series.dataFields.valueY = 'value';
+  series.dataFields.dateX = 'year';
+  series.name = description;
```

Comment:
I'm not sure exactly where it is, but I think this is the line that adds the "legend"? I feel like it could be deleted -- since there's just one line on the graph, it's not that relevant or helpful to have something that just matches that line to the title of the graph.

Reply:
Totally agree, legend removed